src/main/java/teammates/storage/sqlentity/Student.java

```diff
@@ -0,0 +1,156 @@
+package teammates.storage.sqlentity;
+
+import java.time.Instant;
+import java.util.Objects;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+/**
+ * Represents a Student entity.
+ */
+@Entity
+@Table(name = "Students")
+public class Student extends User {
+    // Cascade?
+    // from hibernate docs: seems like we can omit this
+    // https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#entity-inheritance-joined-table
+    @OneToOne(mappedBy = "id")
+    @Column(nullable = false)
+    private int id;
+
+    // Cascade?
+    @OneToOne
+    @JoinColumn(name = "teamId")
+    private Team team;
```

Comment:
I've decided to move this to User instead. Two reasons, firstly there is a little know "Instructors" team that is hardcoded. Secondly, this gives us the flexibility to support teams for instructors or teams with both student and instructors in the future.

Reply:
Also, this is definitely not OneToOne. This is a ManyToOne relationship.

Reply:
Yeah that makes sense!
> I've decided to move this to User instead. Two reasons, firstly there is a little know "Instructors" team that is hardcoded. Secondly, this gives us the flexibility to support teams for instructors or teams with both student and instructors in the future.

I think I misunderstood the direction. Please correct me if im wrong, from the Student's POV to have this attribute (previously), we read it as `Many Students belong to One Team`? And we should be going for bidirectional r/s too?
> Also, this is definitely not OneToOne. This is a ManyToOne relationship.

Reply:
For this, might want to take a look at what each annotation means and at the differences between them. For best practices, I try to follow the ones defined here: https://thorben-janssen.com/best-practices-many-one-one-many-associations-mappings/.